updating faq (#2103)
elastic/docs-content: deploy-manage/deploy/elastic-cloud/serverless.md

To learn more about serverless status, see [Service status](../../cloud-organization/service-status.md).

## Frequently Asked Questions (FAQ) about Elastic Serverless Projects [general-what-is-serverless-elastic-answers-to-common-serverless-questions]

> The following FAQ addresses common questions about using Elastic serverless projects.

Reviewer: ```suggestion
The following FAQ addresses common questions about using {{serverless-full}} projects.
```

### Pricing and availability

> **Q: Where can I learn about pricing for serverless?**  

Reviewer: ```suggestion
**Q: Where can I learn about pricing for {{serverless-short}}?**  
```

> A: See serverless pricing information for [Search](https://www.elastic.co/pricing/serverless-search), [Observability](https://www.elastic.co/pricing/serverless-observability), and [Security](https://www.elastic.co/pricing/serverless-security).

Reviewer: I think these are the right variables

```suggestion
A: See pricing information for [{{es-serverless}}(https://www.elastic.co/pricing/serverless-search), [{{observability}}](https://www.elastic.co/pricing/serverless-observability), and [{{sec-serverless}}](https://www.elastic.co/pricing/serverless-security).
```

> **Q: In which cloud regions is Elastic’s serverless offering available?**  

Reviewer: {{serverless-full}}

> A: Elastic serverless projects are available in select AWS regions, with plans to expand to more regions and cloud providers. For the latest list, refer to the [Serverless regions](/deploy-manage/deploy/elastic-cloud/regions.md).

Reviewer: no strong opinion about "For the latest list" so will defer to Kaarina. empty link title will adopt the title of the target page 🪄 

```suggestion
A: {{serverless-full}} is available in select AWS, GCP, and Azure regions, with plans to expand to more regions. For more information, refer to [](/deploy-manage/deploy/elastic-cloud/regions.md).
```

### Data management

> **Q: How can I move data to or from serverless projects?**  

Reviewer: ```suggestion
**Q: How can I move data to or from {{serverless-short}} projects?**  
```

> A: We are working on data migration tools! In the interim, [use Logstash](logstash://reference/index.md) with {{es}} input and output plugins to move data to and from serverless projects.

Reviewer: exclamation mark sounds defensive.

```suggestion
A: We are working on data migration tools. In the interim, [use Logstash](logstash://reference/index.md) with {{es}} input and output plugins to move data to and from {{serverless-short}} projects.
```

**Q: Can I request backups or restores for my projects?**  
A: It is not currently possible to request backups or restores for projects, but we are working on data migration tools to better support this.

### Security, compliance, and access

> **Q: How can I create serverless service accounts?**  

Reviewer: ```suggestion
**Q: How can I create {{serverless-full}} service accounts?**  
```

> A: Create API keys for service accounts in your serverless projects. Options to automate the creation of API keys with tools such as Terraform will be available in the future.

Reviewer: ```suggestion
A: Create API keys for service accounts in your {{serverless-short}} projects. Options to automate the creation of API keys with tools such as Terraform will be available in the future.
```

### Project lifecycle and support

> **Q: How does serverless ensure compatibility between software versions?**  

Reviewer: ```suggestion
**Q: How does {{serverless-full}} ensure compatibility between software versions?**  
```

A: Connections and configurations are unaffected by upgrades. To ensure compatibility between software versions, quality testing and API versioning are used.

> **Q: Can I convert a serverless project into a hosted deployment, or a hosted deployment into a serverless project?**  

Reviewer: ```suggestion
**Q: Can I convert a {{serverless-full}} project into an {{ech}} deployment, or a hosted deployment into a {{serverless-short}} project?**  
```

> A: Projects and deployments are based on different architectures, and you are unable to convert.

Reviewer: ```suggestion
A: Projects and deployments are based on different architectures, so you are unable to convert.
```

> **Q: Can I convert a serverless project into a project of a different type?**  

Reviewer: ```suggestion
**Q: Can I convert a {{serverless-short}} project into a project of a different type?**  
```

A: You are unable to convert projects into different project types, but you can create as many projects as you’d like. You will be charged only for your usage.

> **Q: How do I raise a support case for serverless?**  

Reviewer: ```suggestion
**Q: How do I raise a support case for {{serverless-full}}?**  
```

> A: Raise a case for your subscription as you do today. In the body of the case, mention you are working in serverless to ensure appropriate support.

Reviewer: ```suggestion
A: Raise a case for your subscription as you do today. In the body of the case, mention you are working with a {{serverless-short}} project to ensure appropriate support.
```
